manual/README.md

```diff
@@ -27,7 +27,7 @@
     * [⪢ NLP and tokenization]
         * [• Data tokenization](Creating_a_table/NLP_and_tokenization/Data_tokenization.md)
         * [• Supported languages](Creating_a_table/NLP_and_tokenization/Supported_languages.md)
-        * [• CJK](Creating_a_table/NLP_and_tokenization/CJK.md)
+        * [• Languages with continuous scripts](Creating_a_table/NLP_and_tokenization/CJK.md)
```

Comment:
Why don't we rename it from `CJK.md` to smth?

Reply:
Missed it. I'll update the menu too.